packages/requirements.txt

```diff
@@ -1,2 +1,2 @@
-django==1.2 
+django==1.11.29 
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2021-33203   
**Check Name:** django: Potential directory traversal via ``admindocs``   
**Severity:** MEDIUM   
**Fixed Version:** 2.2.24, 3.1.12, 3.2.4   
**Description:** Django before 2.2.24, 3.x before 3.1.12, and 3.2.x before 3.2.4 has a potential directory traversal via django.contrib.admindocs. Staff members could use the TemplateDetailView view to check the existence of arbitrary files. Additionally, if (and only if) the default admindocs templates have been customized by application developers to also show file contents, then not only the existence but also the file contents would have been exposed. In other words, there is directory traversal outside of the template root directories.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2021-33203